.github/workflows/release-generate-ci.yaml

```diff
@@ -60,12 +60,12 @@ jobs:
       - name: Generate CI (on workflow dispatch)
         if: github.event_name == 'workflow_dispatch'
         working-directory: ./src/github.com/openshift-knative/hack
-        run: go run ./cmd/prowcopy --branch ${{ inputs.branch }} --tag ${{ inputs.branch }} --config config/serverless-operator.yaml
+        run: go run ./cmd/prowcopy --branch ${{ inputs.branch }} --tag "${{ inputs.branch }}.0" --config config/serverless-operator.yaml
```

Comment:
When you're changing this, do you plan to re-generate the CI config for release-1.32 branch?

Reply:
No, I don't but this is necessary moving forward to make sure that the Dockerfile olm-catalog/serverless-operator/index/Dockerfile is using the correct versioned image